autosklearn/evaluation/__init__.py

```diff
@@ -234,21 +234,24 @@ def run(self, config, instance=None,
 
     def get_splitter(self, D):
         y = D.data['Y_train'].ravel()
-
+        train_size = 0.67
+        if not self.resampling_strategy_args and self.resampling_strategy_args.get('train_size'):
```

Comment:
Shouldn't this rather be `if self.resampling_strategy_args and self.resampling_strategy_args.get('train_size'):`? 

And how about simplifying it?

```
if self.resampling_strategy_args:
    train_size = self.resampling_strategy_args.get('train_size', train_size)
```